apps/ledger-live-mobile/src/components/RootNavigator/BaseNavigator.tsx

```diff
@@ -264,8 +264,8 @@ export default function BaseNavigator() {
               const onError =
                 route.params?.onError || (route.params as unknown as typeof route)?.params?.onError;
               // @TODO replace with correct error
-              if (onError && typeof onError === "function")
-                onError(route.params.error || new Error("Request account interrupted by user"));
+              if (onError && typeof onError === "function" && route.params.error)
+                onError(route.params.error);
```

Comment:
@Justkant and @hedi-edelbloute, what do you think about this?
```suggestion
              if (onError && typeof onError === "function") {
                onError(route.params.error);
              } else {
                onClose(new Error("Request account interrupted by user"));
              }```

Reply:
`&& route.params.error` seems the more generic approach to me.

Reply:
This change broken the wallet-api